www/main/toolbar/components/Toolbar.jsx

```diff
@@ -8,13 +8,16 @@ const Toolbar = () => {
   const toolbarItems = ['sunder-gutka', 'ceremonies', 'sync-button', 'lock-screen', 'settings'];
 
   return (
-    !minimizedBySingleDisplay && (
-      <div id="toolbar-nav">
-        {toolbarItems.map((itemName, index) => {
-          return <ToolbarItem key={index} itemName={itemName} />;
-        })}
-      </div>
-    )
+    <div
+      id="toolbar-nav"
+      className={`${
+        minimizedBySingleDisplay ? 'single-display-hide-left' : 'single-display-show-left'
```

Comment:
maybe we can shorten the classname here

Reply:
@theamanjs you got any suggestion for a short classname?

Reply:
Since it's all related to transition can we name it as `animate-left-in` `animate-left-out` Though I feel all the minor changes like code improvement can go in different PR